Clarify Amount calculation (#323)
fiskaltrust/interface-doc: doc/middleware-de-kassensichv/data-structures/data-structures.md

| `ProductBarcode`          | `string (50)` | mandatory if applicable           | Use to send the Global Trade Item Number (GTIN) if the charge item represents an article.                                                                                                                                                                                                                                                              | 1.3         |
| `ProductGroup`            | `string (50)` | mandatory if available            | Name of the product group.                                                                                                                                                                                                                                                                                                                             | 1.3         |     
| `Quantity`                | `decimal (3)` | mandatory if available            | Quantity   [Payload and Export Amount Calculation](#Payload-and-Export-Amount-Calculation)   | 1.3         |

> | `Amount`               | `decimal (5)` | 0.0 mandatory            | [see general part](../../general/data-structures/data-structures.md#Payload-and-Export-Amount-Calculation) and [Payload and Export Amount Calculationg](#Payload-and-Export-Amount-Calculation)          | 1.3         |

Reviewer: ```suggestion
| `Amount`               | `decimal (5)` | 0.0 <br/> mandatory            |  Gross total price of service(s) [see general part](../../general/data-structures/data-structures.md) and [Payload and Export Amount Calculation](#Payload-and-Export-Amount-Calculation)          | 1.3         |
```

| `UnitPrice`               | `decimal (5)` | mandatory if available            | Price per unit incl. VAT                                                                                                                                                                                                                                                                                                                               | 1.3         |
| `VATAmount`               | `decimal (5)` | mandatory for special cases       | In some special cases of taxation (e.g. car spare part in the car repair shop), the VAT is not a percentage of the net-price (NETTO) or the gross-price. For these cases, the field  `VATAmount` is mandatory and a `ftChargeItemCase` that maps to DSFinV-K UST_SCHLUESSEL 7 should be used. The value of the field `VATRate` should be set to `0.0`. | 1.3         |

### Payload and Export Amount Calculation

> To get a positive Amount: Amount and Quantity have to be positiv. This is relevant with credit notes including rabatt, which must be positiv.

Reviewer: ```suggestion
To get a positive Amount: Amount and Quantity have to be positive. This is relevant e.g. for credit notes including a discount, which must be positive.
```
